web/templates/presence_row.hbs

```diff
@@ -12,7 +12,7 @@
                 </div>
             </a>
         {{else if user_list_style.WITH_AVATAR}}
-            <div class="user-profile-picture">
+            <div class="user-profile-picture avatar_placeholder">
```

Comment:
This class name should use dashes, and probably should have a more specific name, perhaps `avatar-preload-background`